.github/workflows/deploy-worker.yml

```
@@ -0,0 +1,26 @@
+name: Deploy Tracera Cloudflare Worker
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)